dashboard/app/models/levels/blockly.rb

```diff
@@ -371,6 +371,13 @@ def localized_instructions
     end
   end
 
+  def localized_markdown_instructions
+    if should_localize? && markdown_instructions
+      I18n.t("data.markdown_instructions").
+        try(:[], "#{name}_markdown_instruction".to_sym)
+    end
+  end
```

Comment:
Any way to dedupe this against similar code (extract a helper)?

```ruby
def localized_failure_message_override
  if should_localize? && failure_message_override
    I18n.t("data.failure_message_overrides").
      try(:[], "#{name}_failure_message_override".to_sym)
  end
end
```

Reply:
good call; I'll open up a separate PR